packages/components/src/components/popover/popover.jsx

```diff
@@ -223,6 +227,7 @@ Popover.defaultProps = {
     portal_container: 'deriv_app',
     margin: 0,
     relative_render: false,
+    should_disable_pointer_events: true,
```

Comment:
will this make the hover and click unavailable by default?

Reply:
I don't think we need it as a default `prop` since we're only using it for the `tooltip` in DBot toolbar.

Reply:
I changed the default value, and set the prop only for DBot toolbar popovers, `view notification` and `manage account setting` from Core.